artemis-server/src/main/java/org/apache/activemq/artemis/core/deployers/impl/FileConfigurationParser.java

```diff
@@ -675,6 +675,8 @@ public void parseMainConfig(final Element e, final Configuration config) throws
 
       config.setCriticalAnalyzerPolicy(CriticalAnalyzerPolicy.valueOf(getString(e, "critical-analyzer-policy", config.getCriticalAnalyzerPolicy().name(), Validators.NOT_NULL_OR_EMPTY)));
 
+      config.setPageSyncTimeout(getInteger(e, "page-sync-timeout", config.getJournalBufferTimeout_NIO(), Validators.GE_ZERO));
```

Comment:
I did not see any xsd changes.. or the FileConfiguration parser tests...

Reply:
I'm sorry, I missed to commit the xsd file. I pushed a new commit to add the xsd file and some changes for the configuration tests.